.env.example

```diff
@@ -11,3 +11,8 @@ CHAPTER_EMAIL=ourEmail@placeholder.place
 EMAIL_USERNAME=emailUsernamePlaceholder
 EMAIL_PASSWORD=emailPasswordPlaceholder
 EMAIL_SERVICE=emailServicePlaceholder
+
+GOOGLE_CLIENT_ID=id from applications dashboard
+GOOGLE_CLIENT_SECRET=secret from applicaitons dashboard
+# Will need to be changed for production
+GOOGLE_REDIRECT_URI=http://localhost:5000/auth/google/callback
```

Comment:
also - why does this need to be an environment variable?

Reply:
See my comment above about the other env vars.